### PR TITLE
Update all.json from Allure Security findings

### PR DESCRIPTION
**Pull request by michalisFr**
These are the new sites for today and the corresponding top-level domains added to `all.json`.
Please keep in mind that the script checks against all.json from the master branch. If there are open PRs, there may be duplicates.
Also, please check if the domains added are any well-known that shouldn't be blacklisted at the top level (eg. netlify, ddns, etc) and change these to include the subdomain.

Please check them for eligibility and liveness, add the additional information (urlscan, screenshot) were applicable, and separate them in eligible, non-eligible and dead.
The curator should only copy the "eligible" ones into the spreadsheet.

Below are 50 new scam sites, along with 43 domains added to `all.json`:

| Site | Domain |
|------|--------|
| walletsbridges.com | walletsbridges.com |
| wcaservice.com | wcaservice.com |
| walletprotectauth.com | walletprotectauth.com |
| wallsynchvalidatevd.com | wallsynchvalidatevd.com |
| smartsyncnode.com | smartsyncnode.com |
| walletsconnects.cloud | walletsconnects.cloud |
| vvalletconnect.com | vvalletconnect.com |
| elsalvadorairdrop.com | elsalvadorairdrop.com |
| w.protocolsfix.app | N/A |
| dapps.weyu.pro | weyu.pro |
| wallectsyncfixs.com | wallectsyncfixs.com |
| walletconnect.raymasterbatch.com | raymasterbatch.com |
| onlinewalletclaim.duckdns.org | duckdns.org |
| authenticatedappwallets.com | authenticatedappwallets.com |
| aaa.mydevsite.biz | mydevsite.biz |
| claim.labornyveri.com | N/A |
| apply.labornyveri.com | N/A |
| wallets.appsregistry.io | N/A |
| app-sync.net | app-sync.net |
| versalunigen.buzz | versalunigen.buzz |
| refreshwallet.company | refreshwallet.company |
| validate.openconnect.company | openconnect.company |
| mmxcs.com | mmxcs.com |
| univingen.click | univingen.click |
| unitsgen.buzz | unitsgen.buzz |
| unitsgeu.buzz | unitsgeu.buzz |
| walletdappconnect.app | walletdappconnect.app |
| scannoderestore.com | scannoderestore.com |
| cronos-active.com | cronos-active.com |
| dash.apollovelo.sbs | apollovelo.sbs |
| xxxcinnexxt.online | xxxcinnexxt.online |
| s.chains-sync.live | N/A |
| unitigent.buzz | unitigent.buzz |
| walletconnect-sale.com | walletconnect-sale.com |
| vaultftconnectserver.com | vaultftconnectserver.com |
| site.theappscenter.com | N/A |
| deskniunique.duckdns.org | N/A |
| app.uniqr-c.com | uniqr-c.com |
| dapp-protocol.playmedia.ma | playmedia.ma |
| thewalletconnects.org | thewalletconnects.org |
| w.connect-v2.live | connect-v2.live |
| syncmainaffixs.com | syncmainaffixs.com |
| portal.dappsconnecsion.online | dappsconnecsion.online |
| centralhubcheck.tk | centralhubcheck.tk |
| vault-connectserver.live | vault-connectserver.live |
| wallet.validatetoday.cc | validatetoday.cc |
| dapps.appsconnectme.online | appsconnectme.online |
| multiserverfix.net | multiserverfix.net |
| univasaldapp.co | univasaldapp.co |
| uuniggen.buzz | uuniggen.buzz |

### Eligible
| Site | Urlscan | Screenshot |
|-------|-------|-------|
|aaa.mydevsite.biz|https://urlscan.io/result/50d51a06-1b64-494e-960c-3beb466ba6b6/dom/|https://user-images.githubusercontent.com/14356826/181046769-bd02d4ce-9a16-463c-9cab-1eac941a1a9d.png|

### Non-eligible
| Site |
|-------|
|walletsbridges.com|
|wcaservice.com|
|claim.labornyveri.com|
|apply.labornyveri.com|
|syncmainaffixs.com|

### Dead
| Site |
|-------|
|walletprotectauth.com|
|wallsynchvalidatevd.com|
|smartsyncnode.com|
|walletsconnects.cloud|
|vvalletconnect.com|
|elsalvadorairdrop.com|
|w.protocolsfix.app|
|dapps.weyu.pro|
|wallectsyncfixs.com|
|walletconnect.raymasterbatch.com|
|onlinewalletclaim.duckdns.org|
|authenticatedappwallets.com|
|wallets.appsregistry.io|
|app-sync.net|
|versalunigen.buzz|
|refreshwallet.company|
|validate.openconnect.company|
|mmxcs.com|
|univingen.click|
|unitsgen.buzz|
|unitsgeu.buzz|
|walletdappconnect.app|
|scannoderestore.com|
|cronos-active.com|
|dash.apollovelo.sbs|
|xxxcinnexxt.online|
|s.chains-sync.live|
|unitigent.buzz|
|walletconnect-sale.com|
|vaultftconnectserver.com|
|site.theappscenter.com|
|deskniunique.duckdns.org|
|app.uniqr-c.com|
|dapp-protocol.playmedia.ma|
|thewalletconnects.org|
|w.connect-v2.live|
|portal.dappsconnecsion.online|
|centralhubcheck.tk|
|vault-connectserver.live|
|wallet.validatetoday.cc|
|dapps.appsconnectme.online|
|multiserverfix.net|
|univasaldapp.co|
|uuniggen.buzz|

### CODE REVIEW

**Comment by michalisFr** [comment]
@ical10 This is the list for today

File: all.json [hunk 1]
```diff
@@ -11399,6 +11399,49 @@
...
+		"walletprotectauth.com",
+		"wallsynchvalidatevd.com",
+		"smartsyncnode.com",
+		"walletsconnects.cloud",
+		"vvalletconnect.com",
+		"elsalvadorairdrop.com",
+		"weyu.pro",
+		"wallectsyncfixs.com",
+		"raymasterbatch.com",
+		"duckdns.org",
```
Comment (jacogr) [inline all.json:11414]: This one seem legit.

File: all.json [hunk 2]
```diff
@@ -11399,6 +11399,49 @@
...
+		"walletdappconnect.app",
+		"scannoderestore.com",
+		"cronos-active.com",
+		"apollovelo.sbs",
+		"xxxcinnexxt.online",
+		"unitigent.buzz",
+		"walletconnect-sale.com",
+		"vaultftconnectserver.com",
+		"uniqr-c.com",
+		"playmedia.ma",
```
Comment (jacogr) [inline all.json:11434]: This one seems above-board - were they sub-domain hijacked?

**Review by jacogr — COMMENTED** [review]
It may be simpler to just list the subdomains as-is and add the top-levels as appropriate.

**Comment by ical10** [comment]
Hi @michalisFr , would it be possible next time to order them alphabetically next time? Thanks.

File: all.json (hunk 1 above)
Comment (ical10) [reply all.json:11414]: leaving that one out.

File: all.json (hunk 2 above)
Comment (ical10) [reply all.json:11434]: Going to take that off for now.

**Review by ical10 — APPROVED** [review]

**Pull request merged**